databases/week1/homework.sql

```diff
@@ -0,0 +1,79 @@
+-- Find out how many tasks are in the task table
+SELECT
+  count(*) as totaltasks
+FROM
+  task;
+--Find out how many tasks in the task table do not have a valid due date
+SELECT
+  COUNT(*) as notvalidduedate
+FROM
+  task
+WHERE
+  due_date IS NULL;
+--Find all the tasks that are marked as done
+SELECT
+  *
```

Comment:
Avoid using * everywhere. Use it only if necessary. For example, here you can make a few specific columns selection as
```SELECT title, status. name FROM task
JOIN status ON task.status_id = status.id 
WHERE status.name = 'Done';```